packages/component-library/src/components/plmg-tabs/plmg-tab.tsx

```diff
@@ -0,0 +1,41 @@
+import { Component, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'plmg-tab',
+  shadow: true,
+})
+export class Tab {
+  /**
+   * Define tabs text content.
+   */
+  @Prop() label: string;
+
+  /**
+   * Define tabs icon content.
+   *
+   * When providing an icon name to this prop, the corresponding icon will be displayed.
+   */
+  @Prop() icon: string;
+
+  /**
+   * Define tabs active state
+   *
+   * Allowed values:
+   *   - true
+   *   - false
+   *
+   * Default: false
+   */
+  @Prop() active: boolean = false;
+
+  /**
+   * Define tabs disabled state
+   *
+   * Allowed values:
+   *   - true
+   *   - false
+   *
+   * Default: false
+   */
+  @Prop({ reflect: true }) disabled: boolean = false;
```

Comment:
Ooh I didn't know about `reflect`! 🆒 
How come we are using it here?

Reply:
I initially put it because I wanted to ensure the html explicitly listed the disabled attribute on the DOM, so I could be sure to handle my css selector and accessibility. To be honest I didn't check if it was doing what I wanted and on inspection it isn't working as I had thought. So, it isn't required.

However, this did start me down a rabbit hole of handling the disabled state in an accessible way. [MozDev](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled) suggests the native `disabled` is not the best way to go and instead to use `aria-disabled` to ensure screen readers can access it rather than skip over. However, there are others saying `disabled` is fine. 
I have gone with using `aria-disabled` and I'd like your feedback on that approach.

Reply:
I do like the use of `aria-disabled` here! I tried this out with the built in mac voiceover (CMD+F5) and noticed that it wasn't announcing the disabled tab as "dimmed" though (this is how the screen reader says buttons are disabled). Maybe we just need quotes around where we are setting the attribute so it forces `true` to be a string? 🤔 

PS. If you only use `disabled` and not `aria-disabled`, it doesn't announce or focus the tab at all with the screen reader, it skips over it, which is not ideal. Since we can see a disabled tab as sighted users, screen reader users should have the equivalent experience, which is why aria-disabled seems better here :)

Here's a demo of `aria-disabled` vs `aria-disabled="true"`

https://user-images.githubusercontent.com/9171326/181509076-05b39250-cf65-45c4-a336-7cba4a1d80d1.mov